### ISSUE TITLE
Can the function of the test set be used?

### ISSUE THREAD

**Issue by AIstudentSH** [KaihuaTang/VQA2.0-Recent-Approachs-2018.pytorch#2]
Thank you for the model, which is very effective for novices.
train with the full train&val splite and get the test-std/test-val results，Is it finished now?

**Comment by KaihuaTang** [comment]
pls wait after the ICCV deadline

**Comment by KaihuaTang** [comment]
> Thank you for the model, which is very effective for novices.
> train with the full train&val splite and get the test-std/test-val results，Is it finished now?

I've supported training whole the trainval split and generate result.json file for you to upload to the vqa2.0 online evaluation server. Since 2018, the test server don't distinguish test-std/test-val, even for test-val, you still need to upload the whole test-std result, just follow the new README.md
